src/main/java/com/microsoft/sqlserver/jdbc/Geography.java

```diff
@@ -118,7 +131,7 @@ public static Geography parse(String wkt) throws SQLServerException {
     }
 
     /**
-     * Constructs a Geography instance that represents a Point instance from its X and Y values and an SRID.
+     * Constructor for a Geography instance that represents a Point instance from its X and Y values and an SRID.
```

Comment:
SRID = Spatial Reference System Identifier
Also add to the param tags in all functions